include/boost/graph/vf2_sub_graph_iso.hpp

```diff
@@ -773,6 +779,10 @@ namespace detail
         bool found_match = false;
 
     recur:
+        if (!user_step_callback()) {
```

Comment:
I think this callback should take in whatever data might change between each call to it. Even though in your case you'll ignore it, someone else might have a reason to examine the internals of whatever state is changing. I guess it's just that `s` object?